### ISSUE TITLE
New website for the project

### ISSUE THREAD

**Issue by rskikuli** [CityZenApp/CityZenApp.github.io#14]
After the initial launch of the project I would propose to refresh the website even more especially now that the project has such progress. 
I would propose to proceed with a fork of the website of NewPipe project https://github.com/TeamNewPipe/website. Here is the website https://newpipe.schabi.org/

It is important to have the opinion from @sidorelauku, @xh3n1 and @marianaballa  since they have worked a lot on the website along with other contributors.

**Comment by AnXh3L0** [comment]
Good idea Redon, I find the existing website outdated and it needs much more tweaking, but that would mean double the work; the NewPipe website is awesome in regards to design so I guess it's a good starting point.

**Comment by AnXh3L0** [comment]
I checked the NewPipe website license, it's MIT, so that works perfectly fine for CityZen. You can close this issue @rskikuli.

**Comment by sidorelauku** [comment]
I think that NewPipe website design fits for CityZen. I could also help with implemenation for the new website.

**Comment by rskikuli** [comment]
I found some interesting themes here as well: http://scoopthemes.com/templates/SevenApp-Template/. Have a look.
